### ISSUE TITLE
Running GitHub Actions without MacOS/Windows Certs 

### ISSUE THREAD

**Issue by rosshoyt** [sudara/pamplejuce#15]
I'd like to use the GitHub actions pipeline to build the plugin for VST/AU, and generate installers as well if possible. However I don't yet have the MacOS or Windows certificates that are used in the [GitHub actions script](https://github.com/sudara/pamplejuce/blob/main/.github/workflows/cmake_ctest.yml). Do you recommend an approach for this situation, before the certificates have been obtained?

Thank you!

**Comment by sudara** [comment]
Hi there!

I would simply comment out the calls to code signing and notarization -- on windows that'll pop a malware warning, however on a mac that'll make life pretty frictionfull for anyone else trying to install (including yourself). 

I'll publish a post soon about how to pop the mac cert in, its straightforward.

**Comment by rosshoyt** [comment]
Thanks! As you suggested, commenting out all the calls to code signing and notarization for Windows and Mac worked. I can't speak for Linux, because I removed that pipeline in my repo.

Following this change, I then needed to update the project name in [dmg.json](https://github.com/sudara/pamplejuce/blob/main/packaging/dmg.json) and [installer.iss](https://github.com/sudara/pamplejuce/blob/main/packaging/installer.iss) to the exact same as ProjectName in CMakeLists.txt; after making that change, the Mac/Windows GitHub Actions pipelines succeeded.

I will close this issue now. However, the Windows installer generated by the doesn't seem to work correctly - I will detail this in another issue. Thanks again!
